src/etc/inc/auth.inc

```diff
@@ -474,6 +474,8 @@ function local_user_set(& $user) {
 		$user_shell = "/bin/tcsh";
 	} elseif (userHasPrivilege($user, "user-copy-files")) {
 		$user_shell = "/usr/local/bin/scponly";
+	} elseif (userHasPrivilege($user, "user-copy-files-chroot")) {
```

Comment:
Yep, actually thinking about it the conflicts are somehow legit here (considering the group vs. user privs case).